File: juneleetcodeProblem/Tree/maxElementBT.java
package juneleetcodeProblem.Tree;

public class maxElementBT {
    Node root;

    public int maxElemenet(Node root) {
        int root_val, left, right, max = Integer.MIN_VALUE;
        if (root != null) {
            root_val = root.data;
            left = maxElemenet(root.left);
            right = maxElemenet(root.right);

            if (left > right) {
                max = left;
            } else {

                max = right;
            }
            if (root_val > max) {
                max = root_val;
            }

        }
        return max;
    }

    public static void main(String[] args) {
        maxElementBT tree = new maxElementBT();
        tree.root = new Node(3);
        tree.root.left = new Node(27);
        tree.root.right = new Node(30);
        tree.root.left.left = new Node(4);
        tree.root.left.right = new Node(5000);
        tree.root.right.left = new Node(60);
        tree.root.right.right = new Node(70);
        System.out.println(tree.maxElemenet(tree.root));
    }

}

File: juneleetcodeProblem/Tree/maxelementBT.py
#from NodeTree import Node
class Node:
    def __init__(self,data):
        self.data=data
        self.left=None
        self.right=None
        
    def maxElement(self,root):
        root_val=0
        right=0
        left=0
        max=0;
        if(root!=None):
            root_val=root.data
            
            left=self.maxElement(self,root.left)
            right=self.maxElement(self,root.right)
            if(left>right):
                max=left
            else:
                max=right
                
            if(root_val>max):
                max=root_val
        return max
    
    
root=Node(1)
root.left=Node(2)
root.right=Node(300)
root.left.left=Node(4)
root.left.right=Node(5)
root.right.left=Node(6)
root.right.right=Node(7)
print(Node.maxElement(Node,root))     
                
            
    
    